ecc_enclave/sgxcclib/sgxcclib.h

```diff
@@ -18,7 +18,6 @@
 #define _SGXCCLIB_H_
 
 #include "types.h"
-#include "sgx_attestation_type.h"
```

Comment:
this is a good catch!

Reply:
Yep, clearly neither you nor Bruno did a clean build of the other PR  ... ;-)

Reply:
yeah! +1 for CI build